.env.example

```diff
@@ -81,6 +84,7 @@ DC_REGISTRY_URL=https://dc.dev.argo.cancercollaboratory.org/
 DC_ID=song.collab
 DC_URL=https://song.rdpc.cancercollaboratory.org/
 DC_BATCH_SIZE=50
+DC_GATEWAY_URL=https://api.rdpc.cancercollaboratory.org/graphql
```

Comment:
Followed the same pattern as for the `DC_URL` variable in the .env.example here, giving the example of the prod endpoint.

Reply:
lets change this to the QA URL as the provided default for devs: https://api.rdpc-qa.cancercollaboratory.org/graphql